### PR TITLE
Web-console: fix alerts from lgtm

### PR DESCRIPTION
**Pull request by mcbrewster**
LGTM is reporting a bunch of alerts this should fix a bunch of them. 

- removes un used variables 
- prevents setting state directly 
- removes useless checks

### CODE REVIEW

File: web-console/src/components/show-log/show-log.tsx [hunk 1]
```diff
@@ -21,6 +21,7 @@ import axios from 'axios';
 import copy from 'copy-to-clipboard';
 import React from 'react';
 
+import { Loader } from '..';
```
Comment (vogievetsky) [inline web-console/src/components/show-log/show-log.tsx]: could you not import from `..`

File: web-console/src/views/query-view/column-tree/column-tree.tsx [hunk 2]
```diff
@@ -385,15 +396,9 @@ export class ColumnTree extends React.PureComponent<ColumnTreeProps, ColumnTreeS
       );
     }
 
-    const { columnTree, selectedTreeIndex, expandedNode } = this.state;
-    if (!columnTree) return null;
-    const currentSchemaSubtree =
-      columnTree[selectedTreeIndex > -1 ? selectedTreeIndex : 0].childNodes;
+    console.log(currentSchemaSubtree);
```
Comment (vogievetsky) [inline web-console/src/views/query-view/column-tree/column-tree.tsx]: woops

File: web-console/src/visualization/stacked-bar-chart.tsx [hunk 3]
```diff
@@ -67,9 +67,10 @@ export class StackedBarChart extends React.Component<StackedBarChartProps, Stack
 
   componentWillReceiveProps(nextProps: StackedBarChartProps): void {
     if (nextProps !== this.props) {
```
Comment (vogievetsky) [inline web-console/src/visualization/stacked-bar-chart.tsx]: pretty sure this if statement does nothing

File: web-console/webpack.config.js [hunk 4]
```diff
@@ -19,7 +19,6 @@
 const process = require('process');
 const path = require('path');
 const postcssPresetEnv = require('postcss-preset-env');
-const BundleAnalyzerPlugin = require('webpack-bundle-analyzer').BundleAnalyzerPlugin;
```
Comment (vogievetsky) [inline web-console/webpack.config.js]: rather than totally removing this please tie it to an env variable

**Comment by vogievetsky** [comment]
LGTM after CI 👍

File: web-console/webpack.config.js [hunk 5]
```diff
@@ -124,10 +124,8 @@ module.exports = env => {
       ],
     },
     performance: {
-      hints: false
+      hints: false,
     },
-    plugins: [
-      // new BundleAnalyzerPlugin()
-    ],
+    plugins: process.env.BUNDLE_ANALYZER_PLUGIN === 'TRUE' ? [new BundleAnalyzerPlugin()] : [],
```
Comment (mcbrewster) [inline web-console/webpack.config.js]: @vogievetsky is this what you wanted? I documented it in both the build script and the read me

Comment (vogievetsky) [reply web-console/webpack.config.js]: looks good to me

**Pull request merged**